src/c_interface.rs

```diff
@@ -108,7 +108,7 @@ pub extern "C" fn sysinfo_refresh_process(system: CSystem, pid: PID) {
         let mut system: Box<System> = Box::from_raw(system as *mut System);
         {
             let system: &mut System = system.borrow_mut();
-            system.refresh_process(Pid(pid));
+            system.refresh_process(Pid(pid.try_into().unwrap()));
```

Comment:
For all these cases, `as _` should be enough.

Reply:
Ok, resubmitted.